pre_commit_hooks/detect_aws_credentials.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import argparse
+import ConfigParser
```

Comment:
this one i'll have to play around with, never used six myself but i've seen it all around me. will experiment.